sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -525,7 +526,7 @@ export class FormTrainingClient {
       return {
         resourceId: resourceId,
         resourceRegion: resourceRegion,
-        //expiresOn: new Date(response.expirationDateTimeTicks),
+        expiresOn: new Date(response.expirationDateTimeTicks * 1000), // Convert to ms
```

Comment:
One discussion point is whether we should hide `expirationDateTimeTicks` from the public API surface.  I remember the argument to include it is to be able to pass the CopyAuthorization Json object to other tools/languages without doing any more transformation.  It does look a little redundant to have both properties.  I think Python has `expirationDateTimeTicks` in the dictionary.  .Net and Java have `ToJson()` or similar.

@bterlson  @xirzec  thoughts?

Reply:
Yeah, I considered that, and I think I should probably remove the `extends CopyAuthorizationResultModel` for Preview 5. We also get the `accessToken` property from that inheritance, so I want to at least get this `Date` conversion in for P4, and then I can work in a refactor for P5.